docs/source/specification.md

```diff
@@ -57,9 +57,9 @@ Below we describe each level of the `rawdata` folder hierarchy in more detail. T
 * Session-level folders *must* be prefixed with a key-value pair that is unique for each session. The key *must* be `ses` and the value *must* be numerical, e.g. `ses-01`.
 * Sessions *should* be assigned ascending numerical labels as they are added to the project. The labels *should* be prefixed with an arbitrary number of 0s for consistent indentation and sorting, e.g. `ses-01`, `ses-02`, `ses-03`.
 * Additional key-value pairs with alphanumerical labels *may* be appended after the `ses` key-value pair. For example, dates can be added as follows: `ses-001_date-20230310`. The keys *should* be consistent across subjects.
-* If a `date` field is added, it *should* be in the format `YYYYMMDD`.
```

Comment:
I have a suggestion that combines these 2 points into 1, let me know what you think:

> If a datetime value is added, it *must* be in [ISO8601](https://en.wikipedia.org/wiki/ISO_8601) format `YYYYMMDDTHHMMSS` and *should* be paired with a `datetime` key e.g. `datetime-20231225T133015`. Alternatively, `date` and/or `time` *may* also be specified as separate keys, paired with the corresponding part of the ISO8601-formatted string. i.e. `date-YYYYMMDD` and/or `time-HHMMSS`.

It's admittedly less explicit/legalistic, but may be more legible?

Reply:
Hey @niksirbi thanks I agree it is nicer to combine this into one paragraph vs. three very repetitive paragraphs. I like that paragraph, though I thought it might make it seem like there is some fundamental split between `date`, `time` and `datetime`. How about the below? (maybe the 'respectively' part is hard to read?)

> If a date, time or datetime is added, it _must_ be in [ISO8610](https://en.wikipedia.org/wiki/ISO_8601) format and _should_ be paired with a `date-`, `time-` or `datetime-` key, respectively. For example:
date: `ses-001_date-20250101` (date format YYYYMMDD)
time: `ses-001_time-140101` (time format HHMMSS)
datetime: `ses-001_datetime-20250101T140101` (datetime format YYYYMMDDTHHMMSS)

Reply:
I like that! Go with it.

Reply:
> If a date, time or datetime is added, it must be in ISO8610 format and should be paired with a date-, time- or datetime- key,

@JoeZiminski just a quick typo check that the ISO standard number is wrong in the suggested edit. Just noticed it and thought I should point it out in case it might be included as-is.

Reply:
thanks @glopesdev!